controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
+ */
+module.exports.readAll = async (req, res) => {
+  const allGenres = await Genre.find({});
+
+  if (res) res.json(allGenres);
+  else return allGenres;
+};
+
```

Comment:
JDoc Comment, pls <3 
e.g.:
```suggestion
/**
 * Serves a single movie genre
 * @param {??} req the request
 * @param {??} res the response
 * @return {Promise} genre
 */
```